unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/SceneContextMenuShould.cs

```diff
@@ -21,6 +21,7 @@ public void SetUp()
 
         [Test]
         [Explicit("Not valid for MVP version")]
+        [Category("Explicit")]
```

Comment:
Those explicit tests were being run when pressing the "run all" button in the Unity's Test Runner. So I added them to the proper category to filter them out.